app/frontend/tests/unit/views/Admin.spec.js

```diff
@@ -0,0 +1,18 @@
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import Vuetify from 'vuetify';
+
+import Admin from '@/views/Admin.vue';
+
+const localVue = createLocalVue();
+localVue.use(Vuetify);
+
+describe('Admin.vue', () => {
+  it('renders', () => {
+    const wrapper = shallowMount(Admin, {
+      localVue,
+      stubs: ['ApiTester', 'Clients', 'BaseSecure', 'JWTDialog']
```

Comment:
some of these child components dont exist but we can go back and fix these tests i suppose

Reply:
I'll get a quick fix in for this